gserver/src/steps.handler.ts

```diff
@@ -211,7 +211,7 @@ export default class StepsHandler {
         step = step.replace(/{}/g, '.*');
 
         //Optional Text
-        step = step.replace(/\(([a-z]+)\)/g, '($1)?');
+        step = step.replace(/\(([a-z ]+)\)/g, '($1)?');
```

Comment:
Could you please add some real example to the `should correctly handle built-in transforms` test?

Reply:
@alexkrechik Sure, be happy to. Would you please point me to the correct folder where I should put the test? Thanks.

Reply:
@alexkrechik after pulling this branch and testing it locally, I realized that the current implementation is working properly so I'll close out this PR. Thanks!